ime/app/src/main/java/com/anysoftkeyboard/devicespecific/DeviceSpecificV26.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2021 Daniel Parks
```

Comment:
Please, change the copyright to AnySoftKeyboard. This is according to https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/CONTRIBUTING.md#license